presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -133,6 +136,16 @@
         this.subqueryPlanner = new SubqueryPlanner(analysis, symbolAllocator, idAllocator, lambdaDeclarationToSymbolMap, metadata, session);
     }
 
+    private static String subQueryNotSupportedError(Node node)
```

Comment:
Can we move this as a helper function to `SemanticExceptions` and possibly partly to `SemanticException` as well. So that we don't have to duplicate it in two places.

Reply:
So the concern is it throws RuntimeException rather than SemanticExceptions, and it is a bit strange to make the static method inside SemanticException.

Reply:
it will be a static method in `SemanticExceptions`. Usually we use `s` to distinguish utils vs objects